packages/modules/web_themes/standard/source/src/components/ChargePoint.vue

```diff
@@ -124,14 +131,17 @@ const targetSoc = computed<number | undefined>(() => {
   }
 });
 
+const showSocTargetSlider = computed(() => {
+  return chargeMode.value === 'stop' || chargeMode.value === 'standby'
+    ? false
+    : true;
```

Comment:
Es ist ungewöhnlich, wenn eine Prüfung positiv ist `false` zurückzugeben. In dem Fall sollte lieber das Kriterium so angepasst werden, dass das Ergebnis direkt stimmt.
```suggestion
  return !['stop', 'standby'].includes(chargeMode.value);
```

Reply:
function updated